cmd/dt/unwrap/unwrap.go

```diff
@@ -496,9 +513,11 @@ func NewCmd(cfg *config.Config) *cobra.Command {
 			if err != nil {
 				return err
 			}
+
 			var successMessage = "Helm chart unwrapped successfully"
 			if fullChartURL != "" {
-				successMessage = fmt.Sprintf(`%s: You can use it now by running "helm install %s --generate-name"`, successMessage, fullChartURL)
+				//successMessage = fmt.Sprintf(`%s: You can use it now by running "helm install %s --generate-name"`, successMessage, fullChartURL)
```

Comment:
Somehow the index feature did not work in here as i was not able to pull chart without specifying the version

That's mainly the reason for this change. I am not sure why that